src/style.css

```
@@ -1 +1,10 @@
 /* styles go here */
+.form-field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+fieldset {
```

you already have class fieldset use it instead of tag name
```suggestion
.fieldset {
```